rtos/mbed_lib.json

```diff
@@ -1,6 +1,198 @@
 {
     "name": "rtos",
     "config": {
-        "present": 1
+        "present": 1,
+
+        "taskcnt": {
+            "help": [
+                "Number of concurrent running user threads <1-250>",
```

Comment:
@bogdanm is there a good way to do human-readable multiline strings in json? If this should be a single line with `\n`s, I can change it to such, it just makes the config file a bit unreadable.

Maybe this is a different discussion, but have we considered supporting both mbed_lib.json and mbed_lib.yaml? This sort of help declaration would become trivial without requiring extra list handling after parsing.